server/src/apub/user.rs

```diff
@@ -87,6 +90,14 @@ impl ActorType for User_ {
     )?;
     Ok(())
   }
+
+  fn send_delete(&self, _creator: &User_, _conn: &PgConnection) -> Result<(), Error> {
+    unimplemented!()
+  }
```

Comment:
Well the user doesnt need to send anything here because it doesnt have any followers. So I think this should just be an empty method.